frontend/src/features/leases/detail/styles.ts

```diff
@@ -86,6 +87,10 @@ export const FormDescriptionLabel = styled(Form.Label)`
   font-weight: 700;
 `;
 
+export const TenantNotes = styled(TextArea)`
+  margin-left: 95px;
```

Comment:
Use rem instead of px for everything except for media queries.

Reply:
I'm also not convinced adding a margin here is the best solution. I'll try and have a closer look at this tmrw.

Reply:
Sounds good, will change to rem in the meantime